tests/test_locales.py

```diff
@@ -211,34 +315,110 @@ def test_format_timeframe(self):
         assert self.locale._format_timeframe("years", 12) == "12 años"
 
         assert self.locale._format_timeframe("now", 0) == "ahora"
-        assert self.locale._format_timeframe("seconds", -1) == "1 segundos"
-        assert self.locale._format_timeframe("seconds", -9) == "9 segundos"
-        assert self.locale._format_timeframe("seconds", -12) == "12 segundos"
-        assert self.locale._format_timeframe("minute", -1) == "un minuto"
-        assert self.locale._format_timeframe("minutes", -2) == "2 minutos"
-        assert self.locale._format_timeframe("minutes", -10) == "10 minutos"
-        assert self.locale._format_timeframe("hour", -1) == "una hora"
-        assert self.locale._format_timeframe("hours", -3) == "3 horas"
-        assert self.locale._format_timeframe("hours", -11) == "11 horas"
-        assert self.locale._format_timeframe("day", -1) == "un día"
-        assert self.locale._format_timeframe("days", -2) == "2 días"
-        assert self.locale._format_timeframe("days", -12) == "12 días"
-        assert self.locale._format_timeframe("week", -1) == "una semana"
-        assert self.locale._format_timeframe("weeks", -2) == "2 semanas"
-        assert self.locale._format_timeframe("weeks", -3) == "3 semanas"
-        assert self.locale._format_timeframe("month", -1) == "un mes"
-        assert self.locale._format_timeframe("months", -3) == "3 meses"
-        assert self.locale._format_timeframe("months", -13) == "13 meses"
-        assert self.locale._format_timeframe("year", -1) == "un año"
-        assert self.locale._format_timeframe("years", -4) == "4 años"
-        assert self.locale._format_timeframe("years", -14) == "14 años"
+        assert self.locale._format_timeframe("seconds", -1) == "hace 1 segundos"
```

Comment:
```self.locale._format_timeframe``` doesn't handle the relative logic, hence why these tests are failing. If you wanted to check whether strings are properly being formatted in their future or past form, that would be done with ```self.locale._format_relative```